### PR TITLE
domain capture

### PR DESCRIPTION
**Pull request by Guaris**
Docs on domain capture added to the manage domains doc.

### CODE REVIEW

File: docs/insomnia/manage-domains.md [hunk 1]
```diff
@@ -27,3 +27,13 @@ If you are having trouble verifying the TXT record, please contact your enterpri
 
 {:.alert .alert-primary}
 **Note**: If you only wish to manage domains that can be used in invite controls and are not interested in using the SSO feature, you can skip verification or verify later.
+
+
+## Domain Capture
+
+1. Navigate to the [domain settings](https://app.insomnia.rest/app/domains/list). 
+
+2. Toggle the **Enable** button. You will see a pop-up modal asking you to enable capture.
```
Comment (cloudjumpercat) [inline docs/insomnia/manage-domains.md:36]:
```suggestion
2. Click the **Enable** toggle. You will see a pop-up modal asking you to enable capture. 
```
https://developers.google.com/style/ui-elements#term-toggle

File: docs/insomnia/manage-domains.md [hunk 2]
```diff
@@ -27,3 +27,13 @@ If you are having trouble verifying the TXT record, please contact your enterpri
 
 {:.alert .alert-primary}
 **Note**: If you only wish to manage domains that can be used in invite controls and are not interested in using the SSO feature, you can skip verification or verify later.
+
+
+## Domain Capture
+
```
Comment (cloudjumpercat) [inline docs/insomnia/manage-domains.md:33]:
```suggestion
## Domain Capture

Domain capture allows you to consolidate all user accounts in your Insomnia org into one Enterprise team.

```

File: docs/insomnia/manage-domains.md [hunk 3]
```diff
@@ -27,3 +27,13 @@ If you are having trouble verifying the TXT record, please contact your enterpri
...
+1. Navigate to the [domain settings](https://app.insomnia.rest/app/domains/list). 
+
+2. Toggle the **Enable** button. You will see a pop-up modal asking you to enable capture. 
+
+Once domain capture is enabled, you can see all captured accounts from the [**Licenses** tab](https://app.insomnia.rest/app/enterprise/licenses). All new users with this domain will be captured automatically. From the license page, the owner or co-owner of the organization has the ability to delete any deactivated accounts, these accounts will still be able to log in, but they won't have access to enterprise data.
```
Comment (cloudjumpercat) [inline docs/insomnia/manage-domains.md:38]:
```suggestion
Once domain capture is enabled, you can see all captured accounts in the [**Licenses** tab](https://app.insomnia.rest/app/enterprise/licenses). All new users with this domain will be captured automatically. From the license page, the owner or co-owner of the organization can delete any deactivated accounts. These accounts can still log in, but they won't have access to enterprise data.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
@Guaris Left a few suggestions!

File: docs/insomnia/manage-domains.md [hunk 4]
```diff
@@ -27,3 +27,15 @@ If you are having trouble verifying the TXT record, please contact your enterpri
 
 {:.alert .alert-primary}
 **Note**: If you only wish to manage domains that can be used in invite controls and are not interested in using the SSO feature, you can skip verification or verify later.
+
+
+## Domain Capture
+
+Domain capture allows you to consolidate all user accounts in your Insomnia org into one Enterprise team.
+
+1. Navigate to the [domain settings](https://app.insomnia.rest/app/domains/list).
```
Comment (notjaywu) [inline docs/insomnia/manage-domains.md:36]:
```suggestion
1. Navigate to the [domain settings](https://app.insomnia.rest/app/enterprise/domains/list). 
```

**Review by notjaywu — COMMENTED** [review]
@Guaris left one comment

**Review by notjaywu — APPROVED** [review]

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**